tests/Utilities/utils.py

```diff
@@ -0,0 +1,6 @@
+''' Adds multiple foods into our food storage. '''
+def addMultipleFoods(foodStorage, foods):
+    foodStorage.addFoods(foods)
+
+def isEmptyInventory(foodStorage):
+    return foodStorage.list() == []
```

Comment:
Is this returning an empty list to show that the food storage is empty?

Reply:
This is returning a boolean. For the passed in `foodStorage` object, we are checking if `.list()` returns an empty list. This would be used if, for example, we want to check that our inventory starts off empty when we first create a `foodStorage`.